cloudflare/resource_cloudflare_access_application.go

```diff
@@ -40,17 +40,15 @@ func resourceCloudflareAccessApplicationCreate(ctx context.Context, d *schema.Re
 		EnableBindingCookie:     d.Get("enable_binding_cookie").(bool),
 		CustomDenyMessage:       d.Get("custom_deny_message").(string),
 		CustomDenyURL:           d.Get("custom_deny_url").(string),
-		HttpOnlyCookieAttribute: d.Get("http_only_cookie_attribute").(bool),
 		SameSiteCookieAttribute: d.Get("same_site_cookie_attribute").(string),
 		LogoURL:                 d.Get("logo_url").(string),
 		SkipInterstitial:        d.Get("skip_interstitial").(bool),
 		AppLauncherVisible:      d.Get("app_launcher_visible").(bool),
 		ServiceAuth401Redirect:  d.Get("service_auth_401_redirect").(bool),
 	}
 
-	if value, ok := d.GetOk("http_only_cookie_attribute"); ok {
-		newAccessApplication.HttpOnlyCookieAttribute = value.(bool)
-	}
+	value := d.Get("http_only_cookie_attribute").(bool)
+	newAccessApplication.HttpOnlyCookieAttribute = &value
```

Comment:
we can use the `BoolPtr` helpers from cloudflare-go in here

```suggestion
	newAccessApplication.HttpOnlyCookieAttribute = cloudflare.BoolPtr(d.Get("http_only_cookie_attribute").(bool))
```

Reply:
you could even move this into the block above using those helpers given we set a default now too.